src/Address/Type.php

```diff
@@ -12,13 +12,27 @@ class Type
      *
      * @var int
      */
-    const T_IPv4 = 4;
```

Comment:
This is not fully backward compatible: people may use reflection to extract all the constants defined by a class: adding these new constants would then break their code.
So, I wouldn't accept such a pull request.

Reply:
Hi Michele,

by considering that case, you shouldn't be allowed to add any new constant in general, as it would break the reflection based introspective enumeration. However, by my understanding, adding a new constant is not generally considered a breaking change.

In any case, you might want to consider merging the code in a new release branch, protecting your user base by the reliance on semantic versioning.

Regards,
Adriano

Reply:
Well, I use reflection in a lot of my projects. So, speaking for myself about IPLib, that would break my own code.

Furthermore, I don't think that we should release a new major release only to remove that `T_...` prefix: I think you should live with it :wink:

Reply:
> I think you should live with it wink

I just would like point out that they should break the PSR naming convention, as far as I remember. Perhaps you should reconsider the renaming. In any case, it's up to you.